packages/blocks/src/matrix-question/styles.js

```diff
@@ -10,7 +10,7 @@ export const MatrixTable = styled.div`
 
 export const MatrixCell = styled.div`
 	align-items: center;
-	border: 1px solid;
+	border: 1px solid var( --crowdsignal-forms-question-border-color );
```

Comment:
This is not an issue introduced by those changes, but I think we'll need to handle it now.
Not all themes have the `--crowdsignal-forms-question-border-color` variable defined, causing the border not to be shown.

Reply:
Should we add it? I guess this also affects all other questions since they use the same variable for the wrapper border by default?

Reply:
As we discussed, let's address this in a separate PR.